docs/addons/instana-addon.md

```diff
@@ -0,0 +1,199 @@
+|-------------------------|-----------------------------------------------------|-------------------------------|
+| `agent.endpointHost`                | Instana Agent backend endpoint host                                | https://ingress-red-saas.instana.io/ (US and ROW)                            |
+| `agent.endpointPort`                | Instana Agent backend endpoint port                                | "443"                            |
+| `agent.key`  | Your Instana Agent key             | nil                            |
+| `agent.downloadKey`  | Your Instana Download key             | nil                            |
+| `agent.env`       | Additional environment variables for the agent    | {}                            |
+| `agent.configuration_yaml`       | Custom content for the agent configuration.yaml file    | nil                            |
+| `cluster.name`             | Display name of the monitored cluster    | "Value of zone.name"                     |
+| `zone.name`               | Zone that detected technologies will be assigned to               | nil                     |
+
+## Bootstraping
+Bootstrap your environment with the following command.
+
+```shell
+cdk bootstrap
+```
+
+and finally you can deploy the stack with the following command.
+```shell
+cdk deploy
+```
+
+## Verify the resources
+Run update-kubeconfig command. You should be able to get the command from CDK output message. More information can be found [here](https://aws-quickstart.github.io/cdk-eks-blueprints/getting-started/#cluster-access).
+
+```console
+aws eks update-kubeconfig --name <your cluster name> --region <your region> --role-arn arn:aws:iam::xxxxxxxxx:role/eks-blue1-eksblue1AccessRole32C5DF05-1NBFCH8INI08A
+```
+
+## Testing
+To validate if Instana Agent configured properly in Amazon EKS. You can run the following command after Amazon EKS cluster in deployed and running.
+```shell
+kubectl get pods -n instana-agent
+```
+Output of the above command will be silimar to below one:
+
+```output
+NAMESPACE       NAME                                  				READY   	STATUS    RESTARTS   AGE
+instana-agent   controller-manager-78479cb596-sktg9   	1/1     	Running   					0          56m
+instana-agent   controller-manager-78479cb596-xz8kn   	1/1     	Running   					0          56m
+instana-agent   instana-agent-gsqx8                   				1/1     	Running   					0          56m
+```
+
+Run following command to verify Instana Agent logs
+```shell
+kubectl logs <instana-agent-pod-name> -n instana-agent # Output shows instana agent logs. pod name in this example is instana-agent-gsqx8
+```
+
+Once you see Instana Agent is running in your Amazon EKS Cluster, you can go to Instana Installation (User Interface) to get the APM metrices.
+
+![Instana Metrics](../assets/images/instana-metrics.png)
+
+## Cleanup
+
+To clean up your EKS Blueprints, run the following commands:
+
+```sh
+cdk destroy
+```
+
+## Disclaimer 
```

Comment:
@arpitn2020 Can you elaborate on this disclaimer? I glanced over your repo and I don't observe dependency on `kubeflow`. If  your addon does depend on `kubeflow`, can you explain why you need it? Also applies to your patterns PR.

Reply:
@shapirov103 Sorry, it was typo and i fixed it.
Fixed in pattern as well https://github.com/aws-samples/cdk-eks-blueprints-patterns/pull/115